doc/api.md

```diff
@@ -1,6 +1,6 @@
 # Solana application : Common Technical Specifications
 
-## 1.3.0
+## 1.4.0
```

Comment:
original intent was to have an API changelog here, i.e. new features corresponding to each version. So the number shouldn't be updated here. But I'm not sure if 1.3.0 is going to be released independently. I'm fine if we have only one release that includes both off-chain signature/refactoring and the swap, and call it 1.3.0. If we do both releases that is also fine.

Reply:
If it's ok for you then we'll release 1.3.0 with both features then

Reply:
I've updated the changelog